### ISSUE TITLE
Do some actions after services initialized

### ISSUE THREAD

**Issue by LogansUA** [typestack/typedi#77]
Hello,

I have a bunch of services decorated with `@Service()` decorator and each service can have **public** method decorated with my custom decorator besides other methods.
```ts
// service-a.ts
@Service()
export class ServiceA {
    @CustomDecorator()
    public async foo() {
        return "foo";
    }

    public async bar() {
        return "bar";
    }
}

// service-b.ts
@Service()
export class ServiceB {
    public async foo() {
        return "foo";
    }

    @CustomDecorator()
    public async bar() {
        return "bar";
    }
}
```

I need to somehow, after all of services are initialized, go trough each of them, find those methods and perform some actions and **it should be done after server is started**.

Is there any way to do it?

Thank you.

**Comment by LogansUA** [comment]
Well, I just created custom decorator and assigned it on needed methods and it's not working. 

Decorators is emitted but, since I'm using methods from services, I got the error `Cannot read property 'findOne' of undefined`. **findOne** is the method from my repository which I injecting in service using `@OrmRepository()` decorator.

So I came back to part where initializing of decorators should wait until container is initialized.

Any thoughts? :)
